lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -2827,6 +2836,13 @@ LogicalResult FIRRTLLowering::visitDecl(NodeOp op) {
   if (symName)
     operand = builder.create<hw::WireOp>(operand, name, symName);
 
+  // Move SV attributes.
+  if (auto svAttrs = sv::getSVAttributes(op)) {
+    if (!symName)
+      operand = builder.create<hw::WireOp>(operand, name);
+    sv::setSVAttributes(operand.getDefiningOp(), svAttrs);
+  }
+
```

Comment:
I think I like that this materializes a wire in case there are SV attributes on the node but the node hasn't been turned into a wire yet. Gives the attributes some guarantee to stick around. I don't think we emit these attributes right now from Chisel, so there are probably no surprise wires suddenly appearing in the output, right?

Reply:
Yes, there should be no new wires as there is no user right now.